utils/land-patterns.stanza

```diff
@@ -153,8 +226,8 @@ public defn apply-soldermask (amount:Double, expand?:True|False):
       (s:ChamferedRectangle|DShape|Capsule):
```

Comment:
big TODO here: replace with new expand-by-amount function that is not equivalent to `offset` or `expand` (preserve shape, but increase dimensions additively rather than multiplicatively)